src/joyride/scripts_handler.cljs

```diff
@@ -86,13 +87,19 @@
           relative-path (script-fn+ title base-path scripts-path relative-path)
           function (function))))))
 
+(defn- cljs-snippet-requiring-js [script-path]
+  (let [strip-js-extension (string/replace script-path #"\.js$" "")]
```

Comment:
Maybe use this extension above instead of `(= ext "cljs")`? Perhaps extract in a predicate?